package.json

```diff
@@ -49,7 +49,7 @@
     "test": "npm run test:main && npm run test:fp",
     "test:fp": "node test/test-fp",
     "test:main": "node test/test",
-    "validate": "npm run style & npm run test"
+    "validate": "npm run style & npm run test & wait"
   },
```

Comment:
After spending some time on it, I found this to be an issue related only to npm versions `2.15.*`— versions `≤ 2.14.22` and `≥ 3.0.0` are not affected.